### ISSUE TITLE
não gostei da parte do poeta

### ISSUE THREAD

**Issue by freddybrasileiro** [victoramsantos/victoramsantos#2]

**Comment by amazingustav** [comment]
Tem bastante coisa estranha aí, tipo o ser poeta e um link do github pro proprio github, sendo que já estamos no github. Acho que o botão faz mais sentido ainda do que o "sou poeta"

**Comment by adalbertopita** [comment]
+1

**Comment by thepabloaguilar** [comment]
Concordo com a Issue e com ponto levantado pelo amigo, @amazingustav!!
